rero/rero-ils
# cypress: split commands.js in several files

* Splits commands.js in several files for better readability.
* Adds circulation custom commands.

Co-Authored-by: Alicia Zangger <alicia.zangger@rero.ch>

## Why are you opening this PR?

- Which task/US does it implement?
- Which issue does it fix?

## Dependencies

My PR depends on:

* rero/rero-ils-ui US1394 branch
* ng-core dev branch

## How to test?

You need a bootstrap and a setup, and run the server with FLASK_DEBUG=False poetry run server, otherwise the test will fail.
Check that FLASK_DEBUG=False is not overwritten in invenio.cfg.

To run the tests on command line: poetry run cypress
Or with a GUI: cd tests/e2e/cypress, then npm run cypress. Click on one of the tests on the left of the screen.
The tests are independent and can be run separately.

## Code review check list
